### ISSUE TITLE
Improve Settings panel Look & feel

### ISSUE THREAD

**Issue by kelson42** [kiwix/kiwix-desktop#640]
Like for #412, we should improve the look & feel of the settings panel and it should not rely anymore on vue.js.

It should look like this (only consider the style)
![image](https://user-images.githubusercontent.com/1029718/117576424-3c907980-b0e6-11eb-8e66-e3f5e8ed85ff.png)

and it looks like this:
![image](https://user-images.githubusercontent.com/1029718/117576449-5d58cf00-b0e6-11eb-92c1-01f9f379eb30.png)

**Comment by kelson42** [comment]
@HanYoung-uwu Because you done such a good job on #421, I thought this might be a ticket you would like to consider?

**Comment by HanYoung-uwu** [comment]
I'm unfamiliar with HTML/CSS/JS. :man_shrugging:

**Comment by kelson42** [comment]
@HanYoung-uwu yes, the goal is to remove it and provide a Pure Qt UI instead. We don't like to have a UI partly in Qt and partly in HTML.

**Comment by kelson42** [comment]
@HanYoung-uwu Thank you very much for this PR. This is how it looks lie for me:
![image](https://user-images.githubusercontent.com/1029718/118238750-3eb55800-b499-11eb-8ab1-98e7267e9b54.png)

What shoudl be improved at least is:
* The font for the local server port, is somehow different (smaller?)
* The background color should be set
* Button should be like in the Kiwix Server button
* We should have a look to the two select boxes tosomehow have a "flat" style (but not sure what all has to be done there).
